arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -280,6 +280,34 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/listConnectionStrings": {
+      "post": {
```

Comment:
@vishrutshah - This is the same as StorageAccounts_ListKeys (which is a POST operation).